client/scss/components/_listing.scss

```diff
@@ -388,67 +388,6 @@ ul.listing {
     opacity: 0.7;
   }
 
-  .index {
-    background-color: $color-grey-4;
-
-    .title .title-wrapper {
-      font-size: 1.2em;
-      opacity: 1;
-
-      a {
-        @include transition(opacity 0.2s ease);
-      }
-
-      a:hover {
-        opacity: 0.7;
-      }
-    }
-
-    .actions {
-      margin-top: 1em;
-    }
-
-    .button {
-      background-color: $color-white;
-      color: $color-teal;
-      border-color: rgba(0, 0, 0, 0.35);
-
-      &:hover {
-        color: $color-white;
-        background: $color-teal-darker;
-        border-color: $color-teal-darker;
-      }
-
-      &:active {
-        color: $color-white;
-        background: #333;
-        border-color: #333;
-      }
-
-      &.bicolor {
-        background: $color-teal-darker;
-
-        &:active {
-          color: $color-white;
-          background: #484848;
-          border-color: #333;
-        }
-      }
-    }
-  }
-
-  .indicator {
```

Comment:
A heads up - I we need to keep the indicator and closing bracket underneath.

I have fixed this up and will push a new update though.